core/src/main/java/org/opensearch/sql/analysis/ExpressionAnalyzer.java

```diff
@@ -185,11 +185,16 @@ public Expression visitConstantFunction(ConstantFunction node, AnalysisContext c
 
   @Override
   public Expression visitFunction(Function node, AnalysisContext context) {
-    FunctionName functionName = FunctionName.of(node.getFuncName());
-    List<Expression> arguments =
-        node.getFuncArgs().stream()
-            .map(unresolvedExpression -> analyze(unresolvedExpression, context))
-            .collect(Collectors.toList());
+    FunctionName functionName;
+    if (node.getIsTableFunction()) {
+      functionName =
+          FunctionName.of(node.getFuncName().substring(node.getFuncName().indexOf(".") + 1));
+    } else {
+      functionName = FunctionName.of(node.getFuncName());
+    }
```

Comment:
Ideally `visitFunction` should be the same as before. If this is necessary, probably you need to implement `visitTableFunction` instead?